### ISSUE TITLE
Health check fails with custom build

### ISSUE THREAD

**Issue by ianwremmel** [GoogleCloudPlatform/continuous-deployment-on-kubernetes#36]
Is there something in https://storage.googleapis.com/solutions-public-assets/jenkins-cd/jenkins-home-v2.tar.gz to make kubernetes health checks work?

I've tried switch to my own customized jenkins docker image along with backing jenkins_home with a perisistent volume claim. Health checks fail, but if i port forward to the pod, jenkins appears to be running correctly. 

At this point, the only theory I have is that health checks are failing for some unknown reason, resulting in 502s when I visit the ingress.

**Comment by viglesiasce** [comment]
Hey @ianwremmel!

Did you disable anonymous viewing? If so you can change the health check to be /login which does return a 200 even in that case.

**Comment by ianwremmel** [comment]
yep. that's what i did. makes perfect sense. thanks! I've been trying to figure this out for three days :)

It might be handy to add a section to the readme about how to customize jenkins instead of just using the provided image.

**Comment by ianwremmel** [comment]
got it working!

once you pointed out the /login part, changing that made perfect sense. I also had to change the port number in the health check from the randomly assigned one to 8080. Any idea why the random port number works with the lab but not a custom build?

Thanks!
